Project/Assets/ML-Agents/Editor/Tests/SampleExporter.cs

```diff
@@ -0,0 +1,61 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.iOS;
+
+namespace Unity.MLAgents
+{
+    public class SampleExporter
+    {
+        const string k_SceneFlag = "--mlagents-scene-path";
+
+        public static void ExportCuratedSamples()
+        {
+            var oldBurst = EditorPrefs.GetBool("BurstCompilation");
+            EditorPrefs.SetBool("BurstCompilation", false);
+            try
+            {
+                var args = Environment.GetCommandLineArgs();
+                var scenes = new List<string>();
+                for (var i = 0; i < args.Length - 1; i++)
+                {
+                    if (args[i] == k_SceneFlag)
+                    {
+                        scenes.Add(args[i + 1]);
+                        Debug.Log($"Exporting Scene {scenes.Last()}");
+                    }
+                }
+
+                foreach (var scene in scenes)
+                {
+                    var assets = new List<string> { scene };
+                    var exampleFolderToAdd = Directory.GetParent(Directory.GetParent(scene).FullName).FullName;
```

Comment:
So this goes from e.g. "Assets/ML-Agents/Examples/Basic/Scenes/Basic.unity" to "Assets/ML-Agents/Examples/Basic/"?

Reply:
yes, I had to do this because the asset database can't detect pure C# dependencies.  So I'm trying to add them manually